docs/guides/self-signed-ca-certs.md

```diff
@@ -51,6 +47,3 @@ in an `override.yaml` file would look like this:
     name: ca-bundle-config
     key: ca-bundle
 ```
-
-
-[Install Crossplane]: ../reference/install.md
```

Comment:
Yes, when using `{{<ref ... >}}` shortcode Hugo will find the markdown page that is associated with this path/title and put it in the markdown. The result _should_ be identical to the markdown engine.

The magic is that if the page moves or is renamed, the link breaks. With traditional markdown links this is just a silently broken link. By using the shortcode Hugo will fail to start. This draws attention to update the link.

99% of the time we're not trying to help the person writing the link but trying to notify the person moving pages to update the links pointing to that page.

Reply:
Gotcha, thanks for the explanation! I think this is a reasonable outlook and ultimately will lead to us having higher quality documentation. We are paying a cost in review as folks cannot see the links in the GitHub rendered markdown, but that feels like the right trade-off to make (i.e. let's rely on Hugo machinery to not let us publish instead of human eyes noticing bad links). That being said, I don't think we need the whole Hugo pipeline in this repo necessarily, but we should run at least enough to verify that links are not broken in CI so we can avoid merging bad content.

Reply:
Right now the only thing we have is the content, none of the layout or "shortcodes" that render links like that. Those live exclusively in the Crossplane.io repo ([example](https://github.com/upbound/crossplane.github.io)) . 

Historically CI (for the links) has been done by just running hugo against the complete repo- if it compiles we pass, if it doesn't then there's something wrong (e.g., broken link). To do that we would have to bring in the rest of the hugo files and configurations.

Defining this process doesn't need to be a blocker for this PR but I think we can checkout the main/master branch of crossplane.io and copy the docs from _this_ repo into the clone and run hugo. I don't imagine this will be very complicated.